src/items/functions/item/item_parse.cpp

```diff
@@ -870,6 +871,67 @@ void ItemParse::parseImbuement(const std::string &tmpStrValue, pugi::xml_node at
 	}
 }
 
+void ItemParse::parseAugment(const std::string &tmpStrValue, pugi::xml_node attributeNode, pugi::xml_attribute valueAttribute, ItemType &itemType) {
+	// TODO: ANALYSE IF THE BEST WAY TO ADD THE AUGMENT IS:
+	/*
+	<attribute key="augments" value="1"> -- 1 = ENABLED AND 0 = DISABLE
+		<attribute key="fierce berserk" value="powerful impact" /> -- WHEN NOT PROVIDING VALUE COULD BE A DEFAULT VALUE ?
+		<attribute key="strong ethereal spear" value="life leech">
+			<attribute key="value" value="5" /> -- WHEN NESTING AND ADDING VALUE COULD BE CUSTOMIZED ?
+		</attribute>
+	</attribute>
+	*/
```

Comment:
Have you decided what's the best way? if yes, please remove this comment code.
you can put in config.lua NOTE, the URL to this PR, and you create a nice "Documentation" to more information about this feature.

Reply:
Yes, thanks. I forgot to remove this comment. Could you help me telling me where it's the best place to create a documentation of how to configure augments ?

Reply:
it's a piece of simple information, and your PR is already very complete, you can check in the config.lua.dist the "Vip System" config.

Reply:
> it's a piece of simple information, and your PR is already very complete, you can check in the config.lua.dist the "Vip System" config.

Thanks, documetation added.